draft-ietf-taps-interface.md

```diff
@@ -3476,7 +3475,7 @@ the goal of a Transport Services system is to support
 such mechanisms as a generic service within the transport layer. This enables applications to more dynamically
 benefit from innovations and new protocols in the transport, although it reduces transparency of the
 underlying communication actions to the application itself. The Transport Services API is designed such that protocol and path selection
-can be limited to a small and controlled set if required by the application for functional or security purposes. Further,
+can be limited to a small and controlled set if required by the application to perform a function or to provide security. Further,
```

Comment:
I stumble over the new text here and would suggest to keep the old one.

Reply:
FWIW, I think the very latest version ("if the application requires this or ...") is better than both the original and the update that was discussed here.